apps/ui-kit/src/lib/components/atoms/toggle/Toggle.tsx

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2024 IOTA Stiftung
```

Comment:
can we make these all 2025?
```suggestion
// Copyright (c) 2025 IOTA Stiftung
```